docs/install_guides/installing-openvino-vcpkg.md

```diff
@@ -0,0 +1,95 @@
+# Install OpenVINO™ Runtime via VCPKG {#openvino_docs_install_guides_installing_openvino_vcpkg}
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Windows, Linux, and macOS 
+                 operating systems, using VCPKG.
+
+.. note::
+   
+   Note that the VCPKG distribution:
+
+   * offers C++ API only
+   * does not offer support for GNA and NPU inference
+   * is dedicated to users of all major OSs: Windows, Linux, macOS.
+   * may offer different hardware support under different operating systems.
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`__
+   
+   .. tab-item:: Processor Notes
+      :sync: processor-notes
+   
+      | To see if your processor includes the integrated graphics technology and supports iGPU inference, refer to:
+      | `Product Specifications <https://ark.intel.com/>`__
+
+   .. tab-item:: Software Requirements
+      :sync: software-requirements
+
+      * `vcpkg <https://vcpkg.io/en/getting-started>`__
+
+
+
+Installing OpenVINO Runtime
+###########################
+
+1. Make sure that you have installed VCPKG on your system. If not, follow the 
+   `VCPKG installation instructions <https://vcpkg.io/en/getting-started>`__.
+
+
+2. Install OpenVINO using the following terminal command:
+
+   .. code-block:: sh
+
+      vcpkg install openvino
+
+   VCPKG also enables you to install only selected components, by specifying them in the command.
+   See the list of `available features <https://vcpkg.link/ports/openvino>`__, for example: 
+
+   .. code-block:: sh
+
+      vcpkg install openvino[cpu,ir]
+
+Note that the VCPKG installation means building all packages and dependencies from source, 
+which means the compiler stage will require additional time to complete the process. 
+
+ 
+Congratulations! You've just Installed OpenVINO! For some use cases you may still 
```

Comment:
after installation we can also mention that users can use OpenVINO is their products using:
```
find_package(OpenVINO)
```
and later
```
cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake
```
(copied from here https://vcpkg.io/en/getting-started.html)